.github/workflows/test.yaml

```diff
@@ -133,7 +133,7 @@ jobs:
           DD_SERVICE: weave-python
           DD_ENV: ci
           WEAVE_SENTRY_ENV: ci
-        run: CI=1 WB_SERVER_HOST=http://wandbservice WF_CLICKHOUSE_HOST=weave_clickhouse WEAVE_SERVER_DISABLE_ECOSYSTEM=1 source /root/venv/bin/activate && pip install 'dspy-ai>=0.1.5' && pip install pydantic -U  && cd weave && pytest --job-num=${{ matrix.job_num }} --timeout=90  ./flow ./integrations ./tests ./legacy/ops_arrow ./legacy/ecosystem ./trace_server ./trace --ddtrace --durations=5
+        run: CI=1 WB_SERVER_HOST=http://wandbservice WF_CLICKHOUSE_HOST=weave_clickhouse WEAVE_SERVER_DISABLE_ECOSYSTEM=1 source /root/venv/bin/activate && pip install 'dspy-ai>=0.1.5' && pip install pydantic -U  && cd weave && pytest -m "not skip_sqlite_client" --job-num=${{ matrix.job_num }} --timeout=90  ./flow ./integrations ./tests ./legacy/ops_arrow ./legacy/ecosystem ./trace_server ./trace --ddtrace --durations=5
```

Comment:
follows the pattern of `skip_clickhouse_client` adding costs only works in clickhouse as of now, because of all the json and array functions I do to calculated the costs. 

I probably could add the json stuff to the orm and make it work for both sqlite and clickhouse, but that is not an insignificant lift. For now I thought it better to write two tests, one for sqlite to make sure include_costs does nothing, and another for clickhouse that include_costs adds a cost object